docs/cloud-native-security/cspm-get-started.asciidoc

```diff
@@ -0,0 +1,124 @@
+
+[discrete]
+=== Set up cloud account access
+The CSPM integration requires access to AWS’s built-in https://docs.aws.amazon.com/IAM/latest/UserGuide/access_policies_job-functions.html#jf_security-auditor[`SecurityAudit` IAM policy] in order to discover and evaluate resources in your cloud account. There are several ways to provide access:
+
+* IAM Role (recommended)
+* Direct access keys
+* Temporary security credentials
+* Shared credentials file
+
+Regardless of which option you use, you’ll need to attach AWS’s built-in https://docs.aws.amazon.com/IAM/latest/UserGuide/access_policies_job-functions.html#jf_security-auditor[`SecurityAudit` IAM policy].
+
+[discrete]
+[[cspm-use-irsa]]
+==== Option 1 - [Recommended] Use Kubernetes Service Account to assume IAM role
+
+Leverage IRSA (IAM role to Kubernetes service-account) feature to get temporary credentials and scoped permission.
+From https://aws.github.io/aws-eks-best-practices/security/docs/iam/#iam-roles-for-service-accounts-irsa[EKS Best Practices]:
+
+IRSA is a feature that allows you to assign an IAM role to a Kubernetes service account. It works by leveraging a Kubernetes feature known as Service Account Token Volume Projection. When pods are configured with a Service Account that references an IAM Role, the Kubernetes API server will call the public OIDC discovery endpoint for the cluster on startup. The endpoint cryptographically signs the OIDC token issued by Kubernetes and the resulting token is mounted as a volume. This signed token allows the pod to call the AWS API's associated IAM role. When an AWS API is invoked, the AWS SDK calls `sts:AssumeRoleWithWebIdentity`. After validating the token's signature, IAM exchanges the Kubernetes issued token for a temporary AWS role credential.
+
+To use this, do not fill in any option in the "Setup Access" section. Click *Save and continue*, and follow the https://docs.aws.amazon.com/eks/latest/userguide/iam-roles-for-service-accounts.html[IRSA] guide.
+
+[discrete]
+[[cspm-use-instance-role]]
+==== Option 2 - Use default instance role
+From https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/iam-roles-for-amazon-ec2.html[IAM roles for Amazon EC2]:
+
+When you create an IAM role using the IAM console, the console creates an instance profile automatically and gives it the same name as the role to which it corresponds.
+
+To use this, do not fill in any option in the "Setup Access" section. Click *Save and continue*.
+
+[discrete]
+[[cspm-use-keys-directly]]
+==== Option 3 - Use access keys directly
+Access keys are long-term credentials for an IAM user or the AWS account root user. To use access keys as credentials, you need to provide the `Access key ID` and the `Secret Access Key`.
+
+For more details, refer to https://docs.aws.amazon.com/general/latest/gr/aws-sec-cred-types.html[Access Keys and Secret Access Keys].
+IMPORTANT: You must select "Programmatic access" when creating the IAM user.
```

Comment:
Need an extra blank line for this to render correctly as an AsciiDoc admonition:
```suggestion

IMPORTANT: You must select "Programmatic access" when creating the IAM user.
```